src/lib/chainNode.ts

```diff
@@ -187,14 +187,13 @@ export default class ChainNode {
           if (dispatchError) {
             this.logger.warn('dispatch error %s', dispatchError)
             transactionDbUpdate('failed')
+            unsub()
             if (dispatchError.isModule) {
               const decoded = this.api.registry.findMetaError(dispatchError.asModule)
-              reject(new Error(`Node dispatch error: ${decoded.name}`))
+              throw new Error(`Node dispatch error: ${decoded.name}`)
             } else {
-              reject(new Error(`Unknown node dispatch error: ${dispatchError}`))
+              throw new Error(`Unknown node dispatch error: ${dispatchError}`)
```

Comment:
I think this works in this case. Note there's another rejection on ln 213 that may still be an issue. I'm not sure the most idiomatic thing to do here. Do we:

1. remove the Promise in entirety and change the return type of `submitRunProcess` to just `void`
2. leave this as a `Promise`, restore the old `rejects` on ln 193 and 195 and then add a `catch` call to the returned promise (i.e. the one created on ln 179
3. update all usages of `submitRunProcess` to catch the error

Thoughts

Reply:
@mattdean-digicatapult not sure, without investigation is hard to say what would be the best course of action, since we do fire and forget we won't be able to feedback to the client for sure so I guess it's matter of just catching and handling in`chainNode.ts`

In this instance we throw and allow express to handle and not block. I would say update other instances to `throw`, so we can log it and .... do something very exciting. Reject won't work because we use it in the controller where we do not await for the actual res from a node. it would be nice to notify end-user tho, not sure how yet.